README.md

```diff
@@ -228,7 +228,3 @@ Here is the corresponding Bibtex entry
 ### Convergence test
 
 * ["The graphical lasso: New Insights and alternatives"](https://web.stanford.edu/~hastie/Papers/glassoinsights.pdf) Mazumder and Hastie, 2012.
-
-### Repeated KFold cross-validation
-
-* ["Cross-validation pitfalls when selecting and assessing regression and classification models"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3994246/) D. Krstajic, L. Buturovic, D. Leahy, and S. Thomas, 2014.
```

Comment:
I like to leave this reference on why use repeatedKFold in. I find that several folks don't know that this is necessary, so it explains our preferred approach.